packages/app/src/sandbox/eval/transpilers/babel/ast/convert-esmodule.ts

```diff
@@ -176,23 +176,47 @@ export function convertEsModule(ast: ESTreeAST): void {
     const statement = program.body[i];
 
     if (statement.type === n.ExportAllDeclaration) {
-      // export * from './test';
-      // TO:
-      // const _csb = require('./test');
-      // Object.keys(_csb).forEach(key => {
-      //   if (key === 'default' || key === '__esModule')
-      //     return;
-      //   exports[key] = _csb[key])
-      // }
+      // Handles "export * from './something';"
 
       addEsModuleSpecifier();
-      const { source } = statement;
+      const { source, exported } = statement;
       if (typeof source.value !== 'string') {
         continue;
       }
+
       const varName = getVarName(`$csb__${generateVariableName(source.value)}`);
       addNodeInImportSpace(i, generateRequireStatement(varName, source.value));
-      program.body.push(generateAllExportsIterator(varName));
+      if (exported) {
+        // export * as test from './test';
```

Comment:
This has changed, not sure yet how this works exactly, will figure this out tomorrow morning